db/migrate/20240405155900_add_category_to_songs.rb

```diff
@@ -0,0 +1,5 @@
+class AddCategoryToSongs < ActiveRecord::Migration[7.1]
+  def change
+    add_column :songs, :category, :string
```

Comment:
Does this just default category to an empty string or NULL?

Reply:
yes, this will initialize the new column with all NULLs